adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceRtbRewardedAd.m

```diff
@@ -0,0 +1,166 @@
+//
+//  GADMAdapterIronSourceRtbRewardedAd.m
+//  ISMedAdapters
+//
+//  Created by Jonathan Benedek on 13/08/2024.
+//  Copyright © 2024 ironSource Ltd. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "GADMAdapterIronSourceRtbRewardedAd.h"
+#import "GADMAdapterIronSourceUtils.h"
+#import "GADMAdapterIronSourceConstants.h"
+
+@interface GADMAdapterIronSourceRtbRewardedAd ()
+
+
+@end
+
+@implementation GADMAdapterIronSourceRtbRewardedAd
+
+
+- (void)loadRewardedAdForConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
+                     completionHandler:
+(GADMediationRewardedLoadCompletionHandler)completionHandler {
+    _rewardedAdLoadCompletionHandler = completionHandler;
+    
+    NSDictionary *credentials = [adConfiguration.credentials settings];
+    NSString *applicationKey = credentials[GADMAdapterIronSourceAppKey];
+    
+    if (applicationKey != nil && ![GADMAdapterIronSourceUtils isEmpty:applicationKey]) {
+        applicationKey = credentials[GADMAdapterIronSourceAppKey];
+    } else {
+        NSError *error = GADMAdapterIronSourceErrorWithCodeAndDescription(
+                                                                          GADMAdapterIronSourceErrorInvalidServerParameters,
+                                                                          @"Missing or invalid IronSource application key.");
+        
+        _rewardedAdLoadCompletionHandler(nil, error);
+        return;
+    }
```

Comment:
same comment about applicationKey as in GADMAdapterIronSourceRtbInterstitialAd.m

Reply:
Done